#! /bin/sh
patch -p0 -f $* < $0
exit $?

Fix connection to servers not implementing random padding

Index: src/data-types/mailstream_ssl.c
===================================================================
RCS file: /cvsroot/libetpan/libetpan/src/data-types/mailstream_ssl.c,v
retrieving revision 1.69
retrieving revision 1.71
diff -u -p -u -r1.69 -r1.71
--- src/data-types/mailstream_ssl.c	21 Nov 2008 17:29:43 -0000	1.69
+++ src/data-types/mailstream_ssl.c	12 Jun 2009 07:28:38 -0000	1.71
@@ -460,7 +460,7 @@ static struct mailstream_ssl_data * ssl_
   gnutls_cipher_set_priority (session, cipher_prio);
   gnutls_kx_set_priority (session, kx_prio);
   gnutls_mac_set_priority (session, mac_prio);
-
+  gnutls_record_disable_padding(session);
   gnutls_dh_set_prime_bits(session, 512);
 
   gnutls_transport_set_ptr(session, (gnutls_transport_ptr) fd);
